src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -0,0 +1,104 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $entity_full_class_name; ?>;
+use <?= $form_full_class_name; ?>;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
```

Comment:
This stuff was actually JUST deprecated from that bundle. Instead, use the `Route` class from the core of Symfony : https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Routing/Annotation/Route.php

There is no `@Method` in the core - instead you should just use the `methods="` option.

Reply:
It was already  done :) I had to test something before push it